Code/GraphMol/MolDraw2D/CMakeLists.txt

```diff
@@ -35,21 +35,47 @@ if(RDK_BUILD_CAIRO_SUPPORT)
   target_compile_definitions(MolDraw2D PUBLIC "-DRDK_BUILD_CAIRO_SUPPORT")
   target_link_libraries(MolDraw2D PUBLIC Cairo::Cairo)
   target_sources(MolDraw2D PRIVATE MolDraw2DCairo.cpp DrawTextCairo.cpp)
+  if(RDK_INSTALL_STATIC_LIBS)
+     target_compile_definitions(MolDraw2D_static PUBLIC "-DRDK_BUILD_CAIRO_SUPPORT")
+     target_link_libraries(MolDraw2D_static PUBLIC Cairo::Cairo)
+     target_sources(MolDraw2D_static PRIVATE MolDraw2DCairo.cpp DrawTextCairo.cpp)
+  endif()
   rdkit_headers(MolDraw2DCairo.h DrawTextCairo.h DEST GraphMol/MolDraw2D)
 endif(RDK_BUILD_CAIRO_SUPPORT)
 
 if(RDK_BUILD_FREETYPE_SUPPORT)
-  find_package(Freetype REQUIRED)
   target_compile_definitions(MolDraw2D PUBLIC "-DRDK_BUILD_FREETYPE_SUPPORT")
-  target_link_libraries(MolDraw2D PUBLIC Freetype::Freetype)
   target_sources(MolDraw2D PRIVATE DrawText.cpp
           DrawTextFT.cpp DrawTextFTSVG.cpp)
+  if(RDK_INSTALL_STATIC_LIBS)
+    target_compile_definitions(MolDraw2D_static PUBLIC "-DRDK_BUILD_FREETYPE_SUPPORT")
+    target_sources(MolDraw2D_static PRIVATE DrawText.cpp
+          DrawTextFT.cpp DrawTextFTSVG.cpp)
+  endif()
+  if( ${CMAKE_SYSTEM_NAME} MATCHES "Emscripten")
```

Comment:
This bit of grossness is required by emscripten, where find_package(Freetype) does not currently work.